### DISCUSSION TITLE
How to make subrows column accessors different than the main?

### DISCUSSION THREAD

**Discussion by TitoSniberb** [TanStack/table#4735] [Q&A]
Let's say my main data objects looks like `[{ a: 'a', b: 'b', subRows: [{ c: 'c', d: 'd' }] }]`

How could I make it so when expanding the subRows, the props `c` and `d` are rendered as accessor keys for the subRows?

```
const columns = [
  {
    accessorKey: 'a'
  },
  {
    accessorKey: 'b'
  },
]
```

**Comment by itshallrun** [comment]
@TitoSniberb have you found an answer to that?
